### PR TITLE
77 add toggle search bar on mobile/hide navbar on scroll

### PR DESCRIPTION
**Pull request by elios11**

### CODE REVIEW

File: src/components/GameDataModal/GameDataModal.jsx
Comment (fgomensoro) [inline src/components/GameDataModal/GameDataModal.jsx:60]: It is not a good practice to manipulate DOM in react. Maybe is there another way to toggle that class when modal opens.

File: src/components/Header/Header.module.css
Comment (fgomensoro) [inline src/components/Header/Header.module.css]: !important is also not a good practice

**Review by fgomensoro — APPROVED** [review]

File: src/components/GameDataModal/GameDataModal.jsx
Comment (elios11) [reply src/components/GameDataModal/GameDataModal.jsx:60]: Sadly, I don't think there's another way of updating the body classlist. I avoided doing that by using ref in most cases, but this is a special one. I'll leave the comment open until I get another approval in case you have a suggestion about it though.

File: src/components/Header/Header.module.css
Comment (elios11) [reply src/components/Header/Header.module.css]: Yup, I think I was testing something and forgot to remove them. Done.

**Review by valselviz — APPROVED** [review]
Great!

**Pull request merged**
